### PR TITLE
Move HWND fields to a separate file

### PR DESCRIPTION
**Pull request by gpetrou**
## Proposed changes

- Move HWND fields to a separate file.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2485)

### CODE REVIEW

File: src/Common/src/Interop/User32/Interop.SetWindowPos.cs [hunk 1]
```diff
@@ -9,12 +9,6 @@ internal static partial class Interop
 {
     internal static partial class User32
     {
-        public static IntPtr HWND_TOP = (IntPtr)0;
-        public static IntPtr HWND_BOTTOM = (IntPtr)1;
-        public static IntPtr HWND_TOPMOST = (IntPtr)(-1);
-        public static IntPtr HWND_NOTOPMOST = (IntPtr)(-2);
-        public static IntPtr HWND_MESSAGE = (IntPtr)(-3);
-
```
Comment (RussKie) [inline src/Common/src/Interop/User32/Interop.SetWindowPos.cs]: This was done in https://github.com/dotnet/winforms/pull/1914/files#diff-f9b6723d928d9514937f72414c606ccc

@hughbe was there a specific reason why you did it like this?

File: src/Common/src/Interop/User32/Interop.HWND.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        public class HWND
```
Comment (hughbe) [inline src/Common/src/Interop/User32/Interop.HWND.cs]: Public static class

File: src/Common/src/Interop/User32/Interop.SetWindowPos.cs (hunk 1 above)
Comment (hughbe) [reply src/Common/src/Interop/User32/Interop.SetWindowPos.cs]: Actually no. I prefer this method in this PR

**Comment by RussKie** [comment]
Thank you @gpetrou but we will keep the existing organisation of the code here.

**Comment by gpetrou** [comment]
@RussKie I am a bit confused. DPI_AWARENESS_CONTEXT for example is on a separate file, so why not this as well? I did it for consistency.

**Comment by weltkante** [comment]
Not speaking for @RussKie but I agree with the decision, these values are _NOT_ window handles so putting them into a `HWND` class makes it look like you could use them wherever a `HWND` is accepted, which is not the case, they are special values only accepted by some APIs.

**Comment by RussKie** [comment]
Thank you @weltkante, you articulated it very well, better that I could 👍

**Pull request closed without merging**